src/content/en/2022/third-parties.md

```diff
@@ -1,20 +1,20 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Third Parties
-description: Third Parties chapter of the 2022 Web Almanac covering third-party impact on web performance and the usage of best practices related to third parties.
+description: This chapter focuses on the topic of how first- and third-party developers are eliminating the third-party negative impact on web performence and if the best practices are followed.
 authors: [imeugenia]
 reviewers: [tunetheweb, kevinfarrugia, alexnj]
 analysts: [kevinfarrugia]
 editors: [shantsis]
 translators: []
 results: https://docs.google.com/spreadsheets/d/1YqoRRsyiNsrEabVLu2nRU98JIG_0zLLuoQhC2nX8xbM/
-featured_quote: Third parties are often associated with negative impact on web performance. However, both third parties and website developers can eliminate it so that third-party functionality isn't harming user experience.
+featured_quote: Third parties are often associated with negative impact on web performance. However, both third-party and website developers can eliminate it so that third-party functionality isn't harming user experience.
 featured_stat_1: 94%
 featured_stat_label_1: Websites using at least one third party
-featured_stat_2: 45%
-featured_stat_label_2: Requests made by websites that are third-party requests.
-featured_stat_3: 34%
-featured_stat_label_3: Third-party requests serving script content.
+featured_stat_2: 60%
+featured_stat_label_2: Top 10 third parties are blocking main thread on more than 60% of websites
```

Comment:
Took it from this one: https://20220915t083540-dot-webalmanac.uk.r.appspot.com/en/2022/third-parties#fig-7

Reply:
I disagree with this stat. Or at least the way it is worded currently.

60% of the websites—that load those third-parties—_may_ block the main thread on mobile if they are as low powered as the Lighthouse mobile client.

So a few things with this:
- It's not 60% of all websites as the above wording implies, but only 60% of websites that load each of those third parties. Many sites will not load any of those third-parties.
- We cannot definitely say that they are blocking the main thread as Lighthouse is a lab-based test and is run on throttled devices. It's entirely possible that the website that use those third parties are only visited by people with more powerful devices. That's unlikely of course but I worry about the definitiveness of saying they definitely are blocking the main thread.

Personally I think this stats is a little hard to explain as a featured stat given the nuances there. I think I'd pick another stat here. Or pick one of that this a little easier to explain (e.g. "90% - Percentage YouTube use which blocks main thread on mobile devices". Still a little definite for my taste but at least can point to an easy to understand audit that backs this up).

Reply:
That is fair point, thank you! 
I want to choose something that shows the impact on performance but also would fit in one short sentence to explain.
How about this one:
```
featured_stat_2: 2 sec
featured_stat_label_2: Potential median render-blocking time caused by Google Maps
```

Reply:
LGTM!